code/renderers/vue3/template/stories/ReactiveDecorators.stories.ts

```diff
@@ -0,0 +1,163 @@
+import { global as globalThis } from '@storybook/global';
+import { userEvent, within } from '@storybook/testing-library';
+import type { Meta, StoryObj } from '@storybook/vue3';
+import { h } from 'vue';
+import { RESET_STORY_ARGS, STORY_ARGS_UPDATED, UPDATE_STORY_ARGS } from '@storybook/core-events';
+import Reactivity from './Reactivity.vue';
+
+const meta = {
+  component: Reactivity,
+  args: { label: 'initial' },
+  play: async ({ canvasElement, id, args }) => {
+    const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+
+    const canvas = within(canvasElement);
+
+    await channel.emit(RESET_STORY_ARGS, { storyId: id });
+    await new Promise((resolve) => channel.once(STORY_ARGS_UPDATED, resolve));
+
+    const input = await canvas.findByLabelText<HTMLInputElement>('Some input:');
+    await userEvent.type(input, 'value');
+
+    await channel.emit(UPDATE_STORY_ARGS, {
+      storyId: id,
+      updatedArgs: { label: 'updated label' },
+    });
+    await new Promise((resolve) => channel.once(STORY_ARGS_UPDATED, resolve));
+  },
+} satisfies Meta<typeof Reactivity>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const NoDecorators: Story = { args: { label: 'No decorators' } };
+
+/**
+ * Decorator should either be a function that returns a VNode or a component
+ * so using VNode is not recommended to wrap the story if you want to use args in reactive mode
+ */
+export const DecoratorVNode: Story = {
+  args: { label: 'Decorator not using args' },
+  decorators: [
+    (storyFn, context) => {
+      return h('div', h(storyFn(context.args)));
```

Comment:
I think I originally wrote this test without passing in context.args. 
I think passing in context.args should not be required right?

Also `storyFn` should a function accepting context and not args right? cc @tmeasday

Reply:
yes i removed it, i was just for some testing purpose